docs/use_cases/recSys-basic.ipynb

```diff
@@ -0,0 +1,3606 @@
+    "news_articles = news_articles[news_articles['date'] >= pd.Timestamp(2018, 1, 1)]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "953ee2e3",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(8583, 6)"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "news_articles.shape"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d2f48988",
+   "metadata": {},
+   "source": [
+    "We will also remove news with short headlines (shorted than 7 words)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "405843be",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "8429"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "news_articles = news_articles[news_articles['headline'].apply(lambda x: len(x.split()) > 6)]\n",
+    "news_articles.shape[0]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "fa28b0ac",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Total number of articles after removing duplicates: 8384\n"
+     ]
+    }
+   ],
+   "source": [
+    "# drop duplicates\n",
+    "news_articles = news_articles.sort_values('headline', ascending=False).drop_duplicates('headline', keep=False)\n",
+    "print(f\"Total number of articles after removing duplicates: {news_articles.shape[0]}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "a613d5d9",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Total number of articles :  8384\n",
+      "Total number of authors :  876\n",
+      "Total number of categories :  26\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Total number of articles : \", news_articles.shape[0])\n",
+    "print(\"Total number of authors : \", news_articles[\"authors\"].nunique())\n",
+    "print(\"Total number of categories : \", news_articles[\"category\"].nunique())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d4fa9174",
+   "metadata": {},
+   "source": [
+    "## 1. Content-based recommender\n",
+    "\n",
+    "Below we will implement our content-based recommended. The recommended list would be same for all users, and would be displayed under the title \"Similar Articles\"."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d6ac9462",
+   "metadata": {},
+   "source": [
+    "To identify which news articles are similar to a given article, we will obtain embedding of the text associated \n",
+    "with all articles. Once we have the embeddings, we will use cosine similarity to retrieve the most similar\n",
+    "articles. The model we chose is Sentence Transformer."
```

Comment:
SentenceTransformers is not a model, more like a framework to use some models for multi-word text embeddings.

Reply:
Changed to:
 The model we chose is from the Sentence Transformers family, often used for text-embedding tasks.